openmicroscopy/openmicroscopy
# copy paste viewer buttons bugfix

# What this PR does

Fixes bug with Copy and Paste buttons to the left of Full Viewer window: https://trac.openmicroscopy.org/ome/ticket/13299
# Testing this PR
1. Check that Copy and Paste buttons on left of full viewer are working as expected
2. Click 'Edit' in full viewer and check that Copy and Paste buttons in 'Edit' panel still work
3. Quick check that Copy & Paste buttons are also working in Preview panel
